### PR TITLE
[Feat] Relative change with `bitset` API feature #2439 in raft

### PR DESCRIPTION
**Pull request by rhdong**

### CODE REVIEW

File: cpp/cmake/thirdparty/get_raft.cmake [hunk 1]
```diff
@@ -61,8 +61,8 @@ endfunction()
 # To use a different RAFT locally, set the CMake variable
 # CPM_raft_SOURCE=/path/to/local/raft
 find_and_configure_raft(VERSION  ${RAFT_VERSION}.00
-        FORK                     ${RAFT_FORK}
-        PINNED_TAG               ${RAFT_PINNED_TAG}
+        FORK                     rhdong
```
Comment (lowener) [inline cpp/cmake/thirdparty/get_raft.cmake:64]: Switch that back before merging

Comment (rhdong) [reply cpp/cmake/thirdparty/get_raft.cmake:64]: Sure, just for running CI in advance. I will revert it after success.

**Review by lowener — APPROVED** [review]
LGTM

**Pull request merged**